example_data/sample_bundle.json

```
@@ -357,47 +398,60 @@
               "coding": [
                 {
                   "system": "http://www.genenames.org/geneId",
-                  "code": "HGNC:6990"
+                  "code": "HGNC:11111"
                 }
               ]
             },
-            "valueString": "MECP2"
+            "valueString": "SMC1A"
           },
           {
             "code": {
               "coding": [
                 {
                   "system": "http://hl7.org/evidence",
                   "code": "evidence",
-                  "display": "Evidence for classification of variant"
+                  "display": "Evidence for classification of variant1"
```

I don't think our display value should change for each variant, IIRC we discussed this before and it's just slipped back in
```suggestion
                  "display": "Evidence for classification of variant"
```

---

Sorry, that was an error, corrected